addons/purchase/wizard/bill_to_po_wizard.py

```diff
@@ -0,0 +1,74 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models, Command, _
+from odoo.exceptions import UserError
+
+
+class BillToPO(models.TransientModel):
+    _name = 'bill.to.po.wizard'
+    _description = 'Bill to Purchase Order'
+
+    purchase_order_id = fields.Many2one(comodel_name='purchase.order')
+    partner_id = fields.Many2one(comodel_name='res.partner')
+
+    def action_add_to_po(self):
+        aml_ids = [abs(record_id) for record_id in self.env.context.get('active_ids') if record_id < 0]
```

Comment:
The table is a union of po lines and am lines which might have the same id
to differentiate, I use negative ids for the account move lines...
And since you could select both po lines and am lines before clicking 'Add to PO', I need to filter them out.